### PR TITLE
Add learners help page to docs/config.ini

### PR DESCRIPTION
**Pull request by thallada**
The "Help" link in the header was linking to the index of the readthedocs. I added a doc config entry so it would link to the learners-specific help page instead.

@dsjen @ajpal @lamagnifica

### CODE REVIEW

**Comment by dsjen** [comment]
Excellent!  I forgot to mention that the help link is tested in an acceptance test (I think).  You'll need to add something like what is done at [test_course_enrollment_demographics.py#L17](https://github.com/edx/edx-analytics-dashboard/blob/master/acceptance_tests/test_course_enrollment_demographics.py#L17) to [test_course_learners.py](https://github.com/edx/edx-analytics-dashboard/blob/master/acceptance_tests/test_course_learners.py) too.

**Comment by lamagnifica** [comment]
doesn't this complete at least one story?

**Comment by thallada** [comment]
@lamagnifica Yep, I forgot to mention that this completes the [AN-7189](https://openedx.atlassian.net/browse/AN-7189) story.

**Comment by lamagnifica** [comment]
@dsjen and @thallada , @stroilova  has asked that we reconsider where this section appears in the guide. Before you merge, I would like to lead a short parking lot discussion or ad hoc meeting about file placement.

**Comment by dsjen** [comment]
Sounds good, @lamagnifica!  How about tomorrow (6/20) after stand up?

File: docs/config.ini [hunk 1]
```diff
@@ -24,6 +24,7 @@ engagement_video_timeline= engagement/Engagement_Video.html
 enrollment_demographics_age = enrollment/Demographics_Age.html
 enrollment_demographics_education = enrollment/Demographics_Education.html
 enrollment_demographics_gender = enrollment/Demographics_Gender.html
+learners = engagement/learners.html
```
Comment (lamagnifica) [inline docs/config.ini]: learners/Individual_Learners.rst

File: acceptance_tests/test_course_learners.py [hunk 2]
```diff
@@ -9,6 +9,8 @@
 
 @skipUnless(ENABLE_LEARNER_ANALYTICS, 'Learner Analytics must be enabled to run CourseLearnersTests')
 class CourseLearnersTests(CoursePageTestsMixin, WebAppTest):
+    help_path = 'engagement/learners.html'
```
Comment (lamagnifica) [inline acceptance_tests/test_course_learners.py]: learners/Individual_Learners.html (edited)

File: docs/config.ini (hunk 1 above)
Comment (thallada) [inline docs/config.ini]: We want to link to the built HTML, right? `learners/Individual_Learners.html`

Comment (lamagnifica) [inline docs/config.ini]: oops! yes indeed.

**Comment by dsjen** [comment]
👍

**Pull request merged**